openlibrary/i18n/__init__.py

```diff
@@ -8,6 +8,7 @@
 from babel.messages import Catalog
```

Comment:
File permissions have changed on this file; let's change them back.

Reply:
Done.